tests/raft_log_replication.py

```diff
@@ -34,28 +34,15 @@
   | empty <= 0 | top_1 <= 7 | top_2 <= 8 | top_3 <= 9 |     Name <= value
   |____________|____________|____________|____________|
 
- Step 2: Replace log entry 3 with a new entry
-   ____________
-  | 3          |     Log index
-  | 7          |     Log term
-  | top_3 <= 10|     Name <= value
-  |____________|
-
- Step 3: Replace log entries 2 and 3 with new entries
-   ____________ ____________
-  | 2          | 3          |     Log index
-  | 8          | 9          |     Log term
-  | top_2 <= 11| top_3 <= 12|     Name <= value
-  |____________|____________|
 
- Step 4: Add an already existing log entry
+ Step 2: Add an already existing log entry
```

Comment:
It _used_ to exist, since you removed the block where it got added, it doesn't anymore.

Reply:
Solved in branch fix-term commit 8d01fa6